flaskr.py

```diff
@@ -1,30 +1,31 @@
 import os
 import sqlite3
+import fnmatch
 from flask import Flask, request, session, g, redirect, url_for, abort, render_template, flash
 
 app = Flask(__name__)
 app.config.from_object(__name__)
 
 # Load default config and override config from an environment varible
 app.config.update(dict(
-	DATABASE = os.path.join(app.root_path, 'flaskr.db'),
-	SECRET_KEY = 'development key',
-	USERNAME = 'admin',
-	PASSWORD = 'default'
+    DATABASE = os.path.join(app.root_path, 'flaskr.db'),
```

Comment:
The formatting change is not neccessary here as we know that it is done on another branch. In a perfect world, the formatting would be done on the separate branch, so PRs don't get cluttered with irrelevant diffs.

This is just a notice, feel free to keep this formatting change, just don't do such repeated change on multiple branches in the future.